cmd/addimport_test.go

```diff
@@ -119,7 +119,7 @@ func Test_AddImportInteractive(t *testing.T) {
 
 	cmd := createAddImportCmd()
 	HoistRootFlags(cmd)
-	input := []interface{}{1, false, fp, "my import", "barfoo.>", ts.OperatorKeyPath}
+	input := []interface{}{1, false, fp, "my import", "barfoo.>", 0}
```

Comment:
do you have a test that doesn't use 0?

Reply:
yes - the ones that provide a signing key directly.